.github/workflows/types.yml

```diff
@@ -0,0 +1,45 @@
+name: types
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  linux_tests:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: true
+      matrix:
+        php: ['8.0']
+        stability: [prefer-lowest, prefer-stable]
```

Comment:
Do we really need `prefer-lowest` builds for this? With this new action we'll double the amount of builds which seems a little overkill to me.

Reply:
Fixed.